src/Controls/StreetControls.js

```diff
@@ -102,19 +239,123 @@ class StreetControls extends FirstPersonControls {
     }
 
     /**
-     * Move the camera
+     * Method called when user click on the ground.</br>
+     * Note that this funtion contains default values that can be overrided, by overriding this function.
+     *
+     * @param {THREE.Vector3} position - The position
+     */
+    onClickOnGround(position) {
```

Comment:
these functions don't need to be exposed.

Reply:
As I write it on the doc 

"Note that this funtion contains default values that can be overrided, by overriding this function."

That means these methods are the default behavior , and it can be overrided to do specific behavior. So it's designed to be exposed.

Reply:
I can't overload them. How are you doing?

Reply:
```
            class myControl extends itowns.StreetControls {
                onClickOnGround(position) {
                    console.log('user click on ground !');
                }
            }
```